### ISSUE TITLE
Reproduce the results PQ 61.12% on the Cityscapes datasets

### ISSUE THREAD

**Issue by jonghyeong726** [bowenc0221/panoptic-deeplab#115]
Thanks for your works on Panoptic DeepLab. I am very newbie and tried to reproduce the results of panoptic segmentation on Cityscapes datasets with X65-DC5 backbone and the same configs. But I only got PQ (60.04%) which is lower.

## Configs
Backbone = X65-DC5 (pretrained)
Batch size = 32
learning rate = 0.001
Train_Iteration = 60k
CROP SIZE = [512, 1024]
Framework = detectron2

## What I changed
Because of pool_kernel_size error, as in the [comment](https://github.com/facebookresearch/detectron2/issues/2072#issuecomment-754644505), I uncommented the corresponding code in ASPP.py

## Performance
PQ: 60.04
AP: 32.41
mIoU: 79.51

I could reproduce AP, mIoU results. I hope to know if this PQ results makes sense and is contained in expected range. 
Moreover, I used two A100(80G) gpus and it requires total 140G for training 32 batch. Is it right memory size for training panoptic deeplab model?
